ZenPacks/zenoss/OpenStackInfrastructure/modeler/plugins/zenoss/OpenStackInfrastructure.py

```diff
@@ -266,6 +266,9 @@ def collect(self, device, unused):
         result = yield client.cinder_volumes()
         results['volumes'] = result['volumes']
 
+        result = yield client.cinder_volumetypes()
+        results['volumetypes'] = result['volume_types']
```

Comment:
probably should use `result.get('volume_types', None)` or whatever for the default if nothing returns